src/bin/rustup-init.rs

```diff
@@ -59,7 +60,9 @@ fn run_rustup_inner() -> Result<utils::ExitCode> {
 
     // Before we do anything else, ensure we know where we are and who we
     // are because otherwise we cannot proceed usefully.
-    process().current_dir()?;
+    process()
+        .current_dir()
+        .context(RustupError::LocatingWorkingDir)?;
```

Comment:
This is essentially the content of `utils::current_dir()`

Reply:
Ah, indeed, updated.